renovate-presets/default.json

```diff
@@ -3,22 +3,21 @@
   "description": "Default preset for use with Jellyfin's repos",
   "extends": [
     "config:base",
-    "helpers:pinGitHubActionDigests",
-    ":rebaseStalePrs",
+    ":disableDependencyDashboard",
+    ":enableVulnerabilityAlertsWithLabel(security)",
+    ":prHourlyLimit2",
+    ":prConcurrentLimit20",
     ":semanticCommits",
     ":semanticCommitScope(deps)",
+    ":timezone(Etc/UTC)",
     "docker:enableMajor",
     "group:testNonMajor",
-    "group:monorepos"
+    "group:monorepos",
+    "helpers:pinGitHubActionDigests"
```

Comment:
Why do you think version numbers are not enough?

Reply:
that was in the preset b4 this and I dont feel strongly about it, but I get why you would want to use the Git SHA's over a tag that can move at any moment.